### DISCUSSION TITLE
exploring-dotnet-6-part-7-analyzers-for-minimal-apis/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#11] [Announcements]
# exploring-dotnet-6-part-7-analyzers-for-minimal-apis/

In this post I look at the analyzers that come built in with ASP.NET Core, including updates to existing analyzers, and new minimal hosting API analyzers.

https://andrewlock.net/exploring-dotnet-6-part-7-analyzers-for-minimal-apis/

**Comment by septst** [comment]
A little late to this post.

First off, thanks for pointing out the incorrect usage of BuildServiceProvider.

A daft question, can you recommend any online resource for learning more about code analysers for .Net and how to setup them. Thank you!

[a comment by an automated account is not shown]
